src/python/pants/option/global_options.py

```diff
@@ -291,6 +294,10 @@ def register_bootstrap_options(cls, register):
              help='Path to a PEM file containing CA certificates used for verifying secure '
                   'connections to --remote-execution-server and --remote-store-server. '
                   'If not specified, TLS will not be used.')
+    register('--remote-oauth-bearer-token-path', advanced=True,
+             help='Path to a file containing an oauth token to use for grpc connections to '
```

Comment:
Would it be worth writing up a very quick README about how to use these options together to hit GCP/RBE? It's not clear whether they are mutually exclusive.

Reply:
Yes, and in particular how that token end up in that file. 

It may be worth breaking oauth support out into some subsystem (I have a draft change for this for other reasons) and have this name a provider instead of a file? But I'm fine with making that change later and deprecating this at that time.